app/cdap/components/NamespaceAdmin/index.tsx

```diff
@@ -57,10 +57,12 @@ const NamespaceAdmin: React.FC = () => {
 
     eventEmitter.on(globalEvents.NSPREFERENCESSAVED, getPreferences);
     eventEmitter.on(globalEvents.ARTIFACTUPLOAD, getDrivers);
+    eventEmitter.on(globalEvents.CONNECTIONCREATE, getConnections);
```

Comment:
Since all the components are in react do we need this event? Can we have the `CreateConnetionModal` have a callback for onConnectionCreate?